WhirlyGlobeSrc/WhirlyGlobe-MaplyComponent/include/MaplyViewController.h

```diff
@@ -91,11 +97,8 @@
   */
 @interface MaplyViewController : MaplyBaseViewController
 
-/// @brief Initialize as a 3D map.
-- (id)init;
-
-/// @brief Initialize as a 2D map.
-- (id)initAsFlatMap;
+/// @brief Initialize as a flat or 3D map.
+- (id)initWithMapType:(MaplyMapType)mapType;
```

Comment:
It seems like the existing method could stay(and be marked as deprecated?) and just call one of the new methods, to avoid breaking existing code.